.github/workflows/develop_build_actions.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:    
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: windows-latest

```

Comment:
What it is?

Reply:
Workflow setting which enables wpf apps build, Ubuntu does not support building wpf so it was crashing